samples/nrf9160/modem_shell/src/gnss/gnss_shell.c

```diff
@@ -13,6 +13,8 @@
 #include "mosh_print.h"
 #include "gnss.h"
 
+#define AGPS_CMD_LINE_INJECT_MAX_LENGTH 3500
```

Comment:
I would do this like:

```suggestion
#define AGPS_CMD_LINE_INJECT_MAX_LENGTH CONFIG_SHELL_CMD_BUFF_SIZE
```

I.e. to indicate better that there is a dependency.

Reply:
Good point. If somebody decides to reduce CONFIG_SHELL_CMD_BUFF_SIZE, we will not waste heap here. It is also possible to inject A-GPS data in smaller pieces so that would be acceptable too. 

I came up with even more sophisticated way (in case somebody increases CONFIG_SHELL_CMD_BUFF_SIZE even more in the future).

Reply:
Minimum of 3500 and CONFIG_SHELL_CMD_BUFF_SIZE should be fine if small numbers are acceptable. Meaning if CONFIG_SHELL_CMD_BUFF_SIZE is like 512 etc., we'd have a small A-GPS buffer but that should be fine then.